quickstart/sparcv8leon3-gr716-mimas.md

```diff
@@ -0,0 +1,110 @@
+# Running system on <nobr>sparcv8leon3-gr716-mimas</nobr>
+
+These instructions describe how to run Phoenix-RTOS on the LEON3/GR716 emulated
+on Mimas A7 Mini FPGA which is called `sparcv8leon3-gr716-mimas`.
+target. Note that the build artifacts, including the system image should be provided in the `_boot` directory. If you
+have not built the system image yet, please refer to the [Building Phoenix-RTOS image](../building/index.md) section.
+
+## Connecting the board
+
+Connect the board to the computer using a USB cable which provide power to the board (There is pin jumper which
+determinate which power line will be in use).
+
+Communication with the board in this case is provided via USB-UART converters connected to connector `P4`
+
+- pins `3-tx (E13 - red)`&`5-rx (D16 - orange)` are used for interfacing with the `phoenixd` server.
+- pins `7-tx (E15 - yellow)`&`9-rx (F14 - orange)` are used for `psh/plo` console.
+- pins `37-M4`&`49-GND` must be connected via `10k` ohm resistor.
+  
+Mimas pinout diagram:
+![Image](_images/MimasA7_Mini_WD.png)
+
+Source: The MimasA7 Mini board's schematic, available on
+  <https://numato.com/product/mimas-a7-mini-fpga-development-board/>
+
+This is how connected device suppose to looks like:
+![Image](_images/MimasA7_Mini_Connected.jpg)
+
+## Flashing Artix FPGA
+
+All information about preparing and uploading a bit stream for the Mimas A7 Mini can be found in the
+[phoenix-rtos-hdl](https://github.com/phoenix-rtos/phoenix-rtos-hdl/blob/master/leon3-numato-mimas-a7-mini/README.md)
+repository.
+
+## Copying flash image using PHFS (phoenixd)
+
+To flash the disk image on board, first step is to verify which port the `plo` serial interface is connected using the
+following command:
+
+```bash
+ls -l /dev/serial/by-id
+```
+
+The output of this command depends on what interfaces are used. (Easiest way to determinate which one are correct is
+run this command, unplug one of them and again using same command check what port disappear).
+
+To provide the disk image to the bootloader, `phoenixd` has to be launched with the following arguments
+(choose suitable ttys device, in this case, USB-UART converter is connected to pins `3 & 5`):
+
+`-s` parameter in `phoenixd` determines from where phoenixd will share files to the device. To simplify this process,
+we can just move to the desired folder and use `.` to point to the current directory.
+
+```bash
+cd _boot/sparcv8leon3-gr716-mimas
+```
+
+```bash
+sudo ./phoenixd -p /dev/ttyUSB[port] -b 115200 -s .
+```
+
+Launch `picocom` to connect to the board using the following command:
+
+```bash
+picocom --imap lfcrlf -b 115200 -r -l /dev/ttyUSB[port] --send-cmd cat
+```
+
+In a second terminal, change directory to `_boot/sparcv8leon3-gr716-mimas` and launch `phoenixd` server on `ttyUSB1`
+port using the following command:
+
+```bash
+sudo ./phoenixd -p /dev/ttyUSB1 -b 115200 -s .
+```
```

Comment:
Running phoenix-daemon is mentioned 2 times and add note to open picocom in new terminal window.